### DISCUSSION TITLE
Recruiting a team

### DISCUSSION THREAD

**Discussion by vinzvinci** [WebXDAO/start-here#43] [Announcements]
Hey everyone, if you are interested to join our Team, feel free to join and comment below.
> Note: This is a volunteering roles
https://github.com/orgs/web3community/teams

We need:
- Blog Maintainer
> - If you are a content writer, you can handle this repo. https://github.com/web3community/blog

- Education Team
> - If you wanna help us to provide content about the technologies we are focused on, you can maintain this repo. https://github.com/web3community/blockchain-dev-path
> - You don't need to create your own content, you can help us gather the learning resources.

**Comment by suitupstan** [comment]
I would love to become a blog maintainer

**Reply by vinzvinci** [reply]
U are now. The blog repository will archive coz we're using our main website for this

**Reply by suitupstan** [reply]
Please don't, I think we can do more in the repo
